lib/Importer/Caffe2.cpp

```diff
@@ -460,6 +460,27 @@ void caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
     return;
   }
 
+  if (typeName == "Copy") {
```

Comment:
Should we combine all these with MKL ops in a single if? There's zero chance that we will support any of these in the next 3 months.

Reply:
Sure, that makes sense. Are `EnsureCPUOutput` and `EnsureDense` also MLK ops?

Reply:
No, they are not related to MKL.